src/core/tests/Test_StatusHandlerTruncation.py

```diff
@@ -414,26 +401,28 @@ def test_both_assessment_and_installation_with_status_error_truncated(self):
         truncated_substatus_file_data = self.__get_substatus_file_json(self.runtime.execution_config.status_file_path)
 
         # Assert assessment truncated
-        self.__expected_truncated_patch_count = 5
-        self.__assert_patch_summary_from_status(truncated_substatus_file_data, Constants.INSTALLATION, Constants.PATCH_ASSESSMENT_SUMMARY, Constants.STATUS_WARNING, self.__patch_count_assessment, complete_substatus_file_data=complete_substatus_file_data, is_under_internal_size_limit=True, is_truncated=True)
+        self.__assert_patch_summary_from_status(truncated_substatus_file_data, Constants.INSTALLATION, Constants.PATCH_ASSESSMENT_SUMMARY, Constants.STATUS_WARNING, self.__patch_count_assessment, errors_count=1, errors_code=Constants.PatchOperationTopLevelErrorCode.WARNING, complete_substatus_file_data=complete_substatus_file_data, is_under_internal_size_limit=True, is_truncated=True)
 
         # Assert installation truncated status file with multi exceptions
-        self.__expected_truncated_patch_count = 547
         self.__assert_patch_summary_from_status(truncated_substatus_file_data, Constants.INSTALLATION, Constants.PATCH_INSTALLATION_SUMMARY, Constants.STATUS_ERROR, self.__patch_count_installation,
             errors_count=5, errors_code=Constants.PatchOperationTopLevelErrorCode.ERROR, installation_substatus_index=1, complete_substatus_file_data=complete_substatus_file_data, is_under_internal_size_limit=True, is_truncated=True)
 
     def test_truncation_method_time_performance(self):
-        """ Comparing truncation code performance on prior and post on 1000 packages with frequency of 300
-        assert truncation code logic time performance is 30 secs more than current (prior truncation) logic"""
+        """ Comparing truncation code performance on prior and post on 750 packages with frequency of 30
+        assert truncation code logic time performance is 30 secs more than current (prior truncation) logic """
+        start_index = 0
+        end_index = 30
+        patch_count = 350
+        expected_time_performance = 30
```

Comment:
TimePerformanceConfig._blah_

Reply:
file is created in /tests/library

done?